filters/filtertest/filtertest.go

```diff
@@ -60,6 +63,15 @@ func (fc *Context) ParentSpan() opentracing.Span {
 	return opentracing.StartSpan("test_span")
 }
 
+func (fc *Context) Logger() filters.FilterContextLogger {
```

Comment:
This is a test context. I think we should just return log.StandardLogger() for simplicity